terminology_compose.yml

```diff
@@ -4,6 +4,9 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.terminology
+    # Uncomment the line below if using an arm mac. For some reason
+    # metamorphosys isn't working in arm docker images.
+    # platform: 'linux/amd64'
```

Comment:
This lets the build get farther, but unfortunately doesn't seem to actually make it work on an m1.